presto-jdbc/pom.xml

```diff
@@ -93,6 +93,11 @@
             <artifactId>joda-time</artifactId>
```

Comment:
You need to update `maven-shade-plugin` to make sure that no dependencies are leaking to user name space.